src/WebJobs.Extensions.DocumentDB/Config/DocumentDBConfiguration.cs

```diff
@@ -152,6 +176,30 @@ internal string ResolveConnectionString(string attributeConnectionString)
             return _defaultConnectionString;
         }
 
+        internal string ResolveTriggerConnectionString()
```

Comment:
I could fall back to the `ConnectionString`, I just want to avoid people accidentally creating an infinite loop and not realizing why.